src/components/pages/Categories/List.vue

```diff
@@ -5,6 +5,9 @@
       :category="categoryName"
       :access="access"
       :done-message="doneMessage"
+      :error-message="errorMessage"
+      :disabled="!disabled"
```

Comment:
[imo]

disabled が false の時に `:disabled` props が true になることに違和感を感じます。
以下のような記述にできると良いかなと思うので修正してみてください。

```suggestion
      :disabled="disabled"
```

Reply:
7e97335a58b66a1491cae50f84b786f2830d08a3

上記記述で動作するよう修正いたしました